package.json

```diff
@@ -25,16 +26,15 @@
     "pkg-server": "cross-env NEXRENDER_REQUIRE_PLUGINS=1 pkg packages/nexrender-server/package.json --out-path bin/",
     "pkg-prelink": "lerna run --scope @nexrender/core pkg-prelink && lerna run --scope @nexrender/server pkg-prelink",
     "pkg": "npm run pkg-prelink && npm run pkg-cli && npm run pkg-server && npm run pkg-worker && npm run pkg-rename",
-    "test": "npm run lint && lerna bootstrap && node test.js"
+    "test": "npm run lint && mocha"
   },
   "devDependencies": {
+    "chai": "^4.3.10",
     "cross-env": "^5.2.0",
     "eslint": "^8.6.0",
     "lerna": "^3.21.0",
     "mocha": "^7.1.2",
-    "pkg": "^5.8.0"
-  },
-  "dependencies": {
-    "@nexrender/action-webhook": "file:packages/nexrender-action-webhook"
```

Comment:
We added this in [this PR](https://github.com/spbk/nexrender/pull/2)

Ideally we can include this without needing to embed it into the deps of the source library? but I don't have a lot of context on this decision so may need to revisit

I know when I added the `nexrender-action-run-command`, I just added the install line into the build script that each worker image intializes with. This might not be best practice so open to discussion here @kevinjcoleman @cbaclig